titleToNumber.js

```diff
@@ -0,0 +1,38 @@
+const titleToNumber = (columnTitle) => {
+  const alphabet = [
+    "A",
+    "B",
+    "C",
+    "D",
+    "E",
+    "F",
+    "G",
+    "H",
+    "I",
+    "J",
+    "K",
+    "L",
+    "M",
+    "N",
+    "O",
+    "P",
+    "Q",
+    "R",
+    "S",
+    "T",
+    "U",
+    "V",
+    "W",
+    "X",
+    "Y",
+    "Z",
+  ];
+  const { length } = alphabet;
+  let result = 0;
+  let letterIndex = 0;
+  for (let i = columnTitle.length - 1; i >= 0; i -= 1) {
```

Comment:
Typically it's better (more common) to use `i--` instead.

Reply:
Linter does not agree with you)